util.js

```diff
@@ -40,6 +43,18 @@ const getPackageJson = async (packageName, version) => {
   }
 };
 
+const readPackageJson = () => {
+  try {
+    const { dependencies, types } = require(path.join(
```

Comment:
This could be simplified to just `path.resolve('./package.json')`?

Reply:
I suppose it could! Been working on other CLI projects lately so just fell into a pattern here.